AmazonSQS/AmazonSQSJCAAPI/src/main/java/fish/payara/cloud/connectors/amazonsqs/api/inbound/SQSPoller.java

```diff
@@ -59,17 +72,24 @@
  * @author Steve Millidge (Payara Foundation)
  */
 class SQSPoller extends TimerTask {
-    
-    AmazonSQSActivationSpec spec;
-    BootstrapContext ctx;
-    MessageEndpointFactory factory;
-    AmazonSQS client;
+
+    private final AmazonSQSActivationSpec spec;
+    private final BootstrapContext ctx;
+    private final MessageEndpointFactory factory;
+    private final AmazonSQS client;
+    private AmazonS3 s3;
+    private static final String S3_BUCKET_NAME_KEY = "s3BucketName";
+    private static final String S3_KEY_KEY = "s3Key";
```

Comment:
probably just setting single KEY on the name is enough: S3_KEY
```suggestion
    private static final String S3_KEY = "s3Key";
```